compiler/graindoc/docblock.re

```diff
@@ -11,6 +11,27 @@ type t = {
   attributes: list(Comments.Attribute.t),
 };
 
+exception
+  MissingFlag({
```

Comment:
I'm a bit confused -- why do we require the flag? Can we not infer a default from the compiler version?

Reply:
Compiler version doesn't work when external libraries authors are using Grain doc. Their `@since` and `@history` will be specific to their development workflows and library version.